scss/partials/_roster.scss

```diff
@@ -363,3 +363,29 @@ li.jsxc-hide-offline {
     }
   }
 }
+
+.jsxc-bar-tags {
+  line-height: 12px;
+}
+
+.jsxc-bar-tags-item {
+  color: var(--jsxc-color-primary-text);
+  font-size: 10px;
+  margin: 0 3px 0 0;
+  padding: 2px;
+}
+
+.jsxc-bar-tags-item:hover {
+  text-overflow: clip;
+  white-space: normal;
+  word-break: break-all;
+  z-index: 99;
+}
+
+.jsxc-bar--has-unread-msg-blink {
+  animation: blinker 1s linear infinite;
+}
+
+@keyframes blinker {
+  50% { opacity: 0; }
+}
```

Comment:
I guess not everyone is a fan of a blinking icon.

Reply:
maybe a flag for it? because i got some messages that a new message should be more present in roster, if a user did not see the notification

Reply:
i added a new option blinkNewMessages: true = blinking, false = not